.github/workflows/basics.yml

```diff
@@ -0,0 +1,38 @@
+# The Travis CI is our full-fledged UI, but this Github workflow
+# runs some basics (build, lint, diff sample json and unit-core)
+# and it should complete quickly, giving more immediate feedback
+# than waiting for Travis
+
+name: 🧱
+
+on: [push]
+
+jobs:
+  basics:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: 10.x
+
+   # Cache yarn deps. thx https://github.com/actions/cache/blob/master/examples.md#node---yarn
+    - name: Get yarn cache directory path
+      id: yarn-cache-dir-path
+      run: echo "::set-output name=dir::$(yarn cache dir)"
+    - uses: actions/cache@v1
+      id: yarn-cache
+      with:
+        path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+        key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-yarn-
+    - run: yarn --frozen-lockfile
+    - run: yarn build-all
+    - run: yarn diff:sample-json
+    - run: yarn lint
+    - run: yarn unit-core
```

Comment:
maybe add `yarn type-check` as well? Relatively cheap (or run just `tsc -p .` for ~half the price)

Reply:
done